06-classification_continued.Rmd

```diff
@@ -415,27 +507,32 @@ accuracy_vs_k <- ggplot(accuracies, aes(x = neighbors, y = mean)) +
   labs(x = "Neighbors", y = "Accuracy Estimate")
 accuracy_vs_k
```

Comment:
Can we please make this plot show all of the K's?

Reply:
done